internal/cmd/http.go

```diff
@@ -99,6 +101,20 @@ func NewHTTPServer(
 			h.ServeHTTP(w, r)
 		})
 	})
+	r.Use(func(h http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if strings.HasSuffix(r.URL.Path, "/") {
+				// Panic if URL can not be parsed if a trailing slash is trimmed.
+				nurl, err := url.Parse(strings.TrimSuffix(r.URL.String(), "/"))
+				if err != nil {
+					panic(err)
+				}
+
+				r.URL = nurl
+			}
```

Comment:
`strings.TrimSuffix` actually already calls `strings.HasSuffix`.
I think you can boil this all down to just the following (avoids panic too):
```suggestion
			r.URL.Path = strings.TrimSuffix(r.URL.Path)
```

Reply:
@GeorgeMac Thank you George!